src/exampleTypes/ResultTable.js

```diff
@@ -127,6 +127,7 @@ let Header = withStateLens({ popover: false, adding: false, filtering: false })(
       disableFilter,
       disableSort,
       field,
+      sortField,
```

Comment:
pretty sure you can simplify this by changing this line to `sortField = field` and then just having sort field later on